cves/kernel/CVE-2020-24394.yml

```diff
@@ -330,8 +337,10 @@ vouch:
 
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of what your answer was.
-  answer:
-  note:
+  answer: true
+  note: |
+    A solution is worked out through discussion in the bug report and 
```

Comment:
I think this note should be reworded. The passive voice "is worked out" will probably sound better in the active voice. Something like "Multiple people discussed solution ideas through the bug report and agreed on the final solution"

Reply:
Fixed